### PR TITLE
New Admin UI

### PR DESCRIPTION
**Pull request by katieli16**

### CODE REVIEW

File: airplane.py [hunk 1]
```diff
@@ -23,14 +23,10 @@ def index():
         # get all itineraries
         get_itineraries = json.loads(airdb.get_itinerary(1))
 
-        return render_template('db.html', type='user', data=get_itineraries)
+        return #render_template('db.html', type='user', data=get_itineraries)
```
Comment (jefftwin) [inline airplane.py]: is this comment intentional? if it is finalized, please just remove the old code

File: airplane.py [hunk 2]
```diff
@@ -111,13 +167,17 @@ def customer_route():
...
-        request.args.get('weight'))
-
-    return res_body
-
+    i_id = request.args.get('id')
+    # fetch bag by itinerary id
+    if request.method == 'GET':
+        res_body = airdb.get_baggage(i_id)
+        if res_body == 0: abort(404)
+	# add a new bag
```
Comment (jefftwin) [inline airplane.py]: this comment should not be indented

File: airplanedb.py [hunk 3]
```diff
@@ -478,27 +479,28 @@ def populate_db(self):
...
             cursor.execute(add_baggage_query)
             db.commit()
-            baggage = {
-                'baggage_id': cursor.lastrowid,
-                'customer_id': customer_id,
-                'baggage_weight': float(bag_weight)
-            }
-            data = json.dumps(baggage, sort_keys=True, indent=4, separators=(',', ': '))
+            bag_object = {
+                    'bag_id': cursor.lastrowid(),
```
Comment (jefftwin) [inline airplanedb.py]: Don't think there should be brackets after cursor.lastrowid ... It wasn't in my commit... does it work even with the bracket?

File: templates/db.html [hunk 4]
```diff
@@ -1,52 +1,206 @@
...
-</body>
-</html>
\ No newline at end of file
+<!DOCTYPE html>
+<html>
+<head>
+    <title>welcome to AIRDB</title>
+    <link rel="stylesheet" type="text/css" href="../static/style.css" />
+</head>
+<style>
```
Comment (jefftwin) [inline templates/db.html]: This <style> stuff should be in a css file instead of just here

File: airplane.py (hunk 1 above)
Comment (katieli16) [reply airplane.py]: fixed

File: airplane.py (hunk 2 above)
Comment (katieli16) [reply airplane.py]: fixed

File: airplanedb.py (hunk 3 above)
Comment (katieli16) [reply airplanedb.py]: It worked either way for me. I removed it

File: templates/db.html (hunk 4 above)
Comment (katieli16) [reply templates/db.html]: oops

File: templates/main.html [hunk 5]
```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>welcome to AIRDB</title>
+    <link rel="stylesheet" type="text/css" href="../static/style.css" />
+</head>
+<style>
+body {font-family: Arial;}
```
Comment (maknoon) [inline templates/main.html]: all css should be moved to the `static/style.css` file or another css file in the `static` directory, and then referenced using `<link rel='stylesheet'>`

File: templates/db.html [hunk 6]
```diff
@@ -1,52 +1,191 @@
...
+            
+   
+ 	        {% endif %}
+        {% endif %}
+         <form action="/main" method="post">
+            <button>Back to Main Menu</button>
+        </form>
+    {% endif %}
+    {% if type == 'user' %}
+ 		<table>
```
Comment (maknoon) [inline templates/db.html]: change tabs to spaces

**Review by maknoon — CHANGES_REQUESTED** [review]

File: templates/main.html (hunk 5 above)
Comment (katieli16) [reply templates/main.html]: fixed

**Review by maknoon — APPROVED** [review]

**Review by jefftwin — APPROVED** [review]

**Pull request merged**